content/02.introduction.md

```diff
@@ -147,6 +147,20 @@ After a virus enters into a host cell, its antigen is presented by major histoco
 One of the main immune responses that contribute to the onset of Acute Respiratory Distress Syndrome (ARDS) in COVID-19 patients is the cytokine storm, which causes an extreme inflammatory response due to a release of pro-inflammatory cytokines and chemokines by immune effector cells.
 In addition to respiratory distress, this mechanism leads to organ failure and death in severe COVID-19 cases [@doi:10.1016/j.jpha.2020.03.001].
 
+##### Cytokine Storm
+
+<!-- As I am cleaning up the text, I'll move the cytokine storm text here, as right now it's very decentralized and one of our goals is to merge everything together in the intro!-->
+IL-6 promotes the differentiation of activated B cells into immunoglobulin-producing plasma cells [@pmid:9052836] and acts as a growth factor for hybridoma and myeloma cells [@pmid:8431556; @pmid:2104241].
+In addition, IL-6 also induces the differentiation of naïve CD4+ T cells into effector T-cell subsets [@doi:10.1158/2326-6066.CIR-14-0022].
+In this way interleukins regulate both the pro- and anti-inflammatory responses.
```

Comment:
I apologize if this is already somewhere else in the text, and I've missed it. Would it make sense to define pro vs anti-inflammatory response? Since the balance of these two is brought up as being important later.

Reply:
```suggestion
In this way interleukins regulate both the pro- and anti-inflammatory responses.
<!-- Note: make sure these terms are defined!-->
```